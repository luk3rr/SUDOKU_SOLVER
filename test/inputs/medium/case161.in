010409200 600100050 400000630 000000000 090000005 700906000 300040960 000507008 980000000 
