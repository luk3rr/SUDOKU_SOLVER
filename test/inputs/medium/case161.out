Solving the following grid:
0 1 0 | 4 0 9 | 2 0 0 
6 0 0 | 1 0 0 | 0 5 0 
4 0 0 | 0 0 0 | 6 3 0 
------+-------+------
0 0 0 | 0 0 0 | 0 0 0 
0 9 0 | 0 0 0 | 0 0 5 
7 0 0 | 9 0 6 | 0 0 0 
------+-------+------
3 0 0 | 0 4 0 | 9 6 0 
0 0 0 | 5 0 7 | 0 0 8 
9 8 0 | 0 0 0 | 0 0 0 


Solution found :')

5 1 3 | 4 6 9 | 2 8 7 
6 2 8 | 1 7 3 | 4 5 9 
4 7 9 | 2 8 5 | 6 3 1 
------+-------+------
8 3 2 | 7 5 4 | 1 9 6 
1 9 6 | 3 2 8 | 7 4 5 
7 4 5 | 9 1 6 | 8 2 3 
------+-------+------
3 5 7 | 8 4 1 | 9 6 2 
2 6 4 | 5 9 7 | 3 1 8 
9 8 1 | 6 3 2 | 5 7 4 

Algorithm: BFS
Total time: 301 ms
Total expanded states: 49236
